docs/content/en/blog/articles/minio-backup/index.md

```diff
@@ -0,0 +1,268 @@
+
+```
+% kubectl get all -n minio
+NAME                        READY   STATUS    RESTARTS   AGE
+pod/minio-5fd4dd687-gzr8j   1/1     Running   0          109s
+
+NAME            TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+service/minio   ClusterIP   10.96.144.61   <none>        9000/TCP   109s
+
+NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/minio   1/1     1            1           109s
+
+NAME                              DESIRED   CURRENT   READY   AGE
+replicaset.apps/minio-5fd4dd687   1         1         1       109s
+```
+
+Using port forwarding, you can expose access to the MinIO UI in the browser on port 9000:
+
+```
+% kubectl port-forward service/minio 9000:9000 -n minio
+Forwarding from 127.0.0.1:9000 -> 9000
+Forwarding from [::1]:9000 -> 9000
+```
+
+Now you can login to MinIO at [http://localhost:9000](http://localhost:9000) using your install time defined credentials:
+
+![MinIO Login](minio-login.png)
+
+Once logged in, you can see that the k8ssandra-medusa bucket was created and is currently empty:
+
+![K8ssandra Medusa Bucket](k8ssandra-medusa-bucket.png)
+
+# Deploy K8ssandra
+
+Now that MinIO is up and running, you can create a namespace for your K8ssandra installation and create a secret for Medusa to access the bucket. Create a `medusa_secret.yaml` file with the following content:
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+ name: medusa-bucket-key
+type: Opaque
+stringData:
+ # Note that this currently has to be set to medusa_s3_credentials!
+ medusa_s3_credentials: |-
+   [default]
+   aws_access_key_id = minio_key
+   aws_secret_access_key = minio_secret
+
+```
+
+Now create the `k8ssandra` namespace and the Medusa secret with the following commands:
+
+```
+kubectl create namespace k8ssandra
+kubectl apply -f medusa_secret.yaml -n k8ssandra
+```
+
+You should now see the **medusa-bucket-key** secret in the **k8ssandra** namespace:
+
+```
+% kubectl get secrets -n k8ssandra
+NAME                  TYPE                                  DATA   AGE
+default-token-twk5w   kubernetes.io/service-account-token   3      4m49s
+medusa-bucket-key     Opaque                                1      45s
+```
+
+You can then deploy K8ssandra with the following custom values file (all default values will be used if not customized here) :
+
+```
+medusa:
+  enabled: true
+  storage: s3_compatible
+  storage_properties:
+      host: minio.minio.svc.cluster.local
+      port: 9000
+      secure: "False"
+  bucketName: k8ssandra-medusa
+  storageSecret: medusa-bucket-key
+```
+
+Save the above file as k8ssandra_medusa_minio.yaml and then install K8ssandra with the following command:
+
+```
+helm install k8ssandra k8ssandra/k8ssandra -f k8ssandra_medusa_minio.yaml -n k8ssandra
+```
+
+Now wait for the Cassandra cluster to be ready by using the following wait command:
+
+```
+kubectl wait --for=condition=Ready cassandradatacenter/dc1 --timeout=900s -n k8ssandra
```

Comment:
This is a super helpful one-liner. If we make a `cheatsheet` at some point I'd like to make sure this is included.

Reply:
added as a note to #546